README.md

```diff
@@ -3,7 +3,7 @@
 # glamorous
 
 React component styling solved with an elegant ([inspired](#inspiration)) API,
-small footprint (~6kb gzipped), and great performance (via [`glamor`][glamor]).
+small footprint (~7kb gzipped), and great performance (via [`glamor`][glamor]).
```

Comment:
Maybe we should just remove the actual number. It seems to be fluctuating a bit anyway.